terraform/aws/etc/eu-west-2_account.tfvars

```diff
@@ -1,16 +1,25 @@
 ecr_repositories = [
   {
-    name = "OneOneOne",
+    name = "111",
     scan = false,
-    expire_PR_after = 14,
+    expire_PR_after = 10,
     prefix_to_keep  = "master",
     number_to_keep  = 10
   },
   # {
   #   name = "nhais",
   #   scan = false,
-  #   expire_PR_after = 14,
+  #   expire_PR_after = 10,
   #   prefix_to_keep = "develop"
   #   number_to_keep = 10
   # },
-]
\ No newline at end of file
+]
+
+account_cidr_block = "10.10.0.0/16"
+
+jumpbox_allowed_ssh = [
+  "91.222.71.98/32",   # Gdansk VPN
+  "195.89.171.5/32",   # Belfast Office
+  "62.254.63.50/32",   # Belfast VPN #1
+  "62.254.63.52/32"    # Belfast VPN #2
+]
```

Comment:
Do you want to publicise our VPN IP addresses(DDoS)? We can always provide this at the command line or some other way.

Reply:
There will be a separate story to clean-up all semi-secret (not harmful, but uncomfortable to be public) and move them to AWS Secrets